src/report/errors.rs

```diff
@@ -130,13 +85,9 @@ impl Errors {
         let index = (loc, report.key, report.msg.to_string(), loc2, loc3);
         if self.seen.contains(&index) {
             return;
-        } else {
-            self.seen.insert(index);
-        }
-        if !self.will_log(&report.primary().location, report.key) {
-            return;
         }
-        log_report(self, &report);
+        self.seen.insert(index);
+        log_report(self, report);
     }
 
     pub fn log_abbreviated(&mut self, loc: &Loc, key: ErrorKey) {
```

Comment:
This and push_header do need their will_log checked, otherwise users will get a long list of "unused" logs that they didn't ask for

Reply:
Ah yes, this is a slight regression. Although, as I understand it, that whole reporting method is locked behind a CLI flag anyways.

I was planning on seeing if I could bring this method of logging more in line with the standard reports.

Reply:
Ah right. The CLI flag used to use an ignore-key to toggle the behavior :) But that got changed. So it's less important now.

Reply:
The density of this method is the main thing. One per line. And easy to extract just the filenames from the unused files one.